apps/judicial-system/web/src/components/AccordionItems/IndictmentsCaseFilesAccordionItem/IndictmentsCaseFilesAccordionItem.tsx

```diff
@@ -348,9 +350,9 @@ const CaseFile: React.FC<CaseFileProps> = (props) => {
                   <Box
                     display="flex"
                     alignItems="center"
-                    component="button"
+                    component={caseFile.canOpen ? 'button' : undefined}
```

Comment:
Why are we showing a file that can not be open, consider maybe just hiding it? 
Just thinking out loud 🤔

Reply:
We need to show it because the user must be able to see that the file is not available. That makes it possible for the user to remove it and/or replace it with another file.